src/lib/Resources/settings/storage_engines/cache.yml

```
@@ -47,6 +47,7 @@ parameters:
         notification_count: 'nc-%s'
         notification_pending_count: 'npc-%s'
         policy: 'p-%s'
+        relation: 'rel-%s'
```

I would shorten it even more, it doesn't seem to conflict with any other pattern in the form below:
```suggestion
        relation: 're-%s'
```